package.json

```diff
@@ -36,7 +36,9 @@
     "package:win": "npm run package:build && electron-builder --win --x64",
     "package:linux": "npm run package:build && electron-builder --linux",
     "package:mac": "npm run package:build && electron-builder --mac",
-    "package:all": "npm run package:build && electron-builder -mwl"
+    "package:all": "npm run package:build && electron-builder -mwl",
+    "cli:help": "npm run build && electron . -h",
```

Comment:
Why `npm run build` before? I think we should not automatically invoke the build task here (`cli:help` should be consistent with the `cli` script).

PS: if you **really** need to execute a task before launching the help command, you could create a new `pre` script in `package.json`, here for example `precli:help` (just add a `pre` prefix to the script name).